library/src/main/java/org/apache/apex/malhar/lib/utils/serde/GenericSerde.java

```diff
@@ -34,31 +39,60 @@
 @InterfaceStability.Evolving
 public class GenericSerde<T> implements Serde<T>
 {
-  private transient ThreadLocal<Kryo> kryos = new ThreadLocal<Kryo>()
```

Comment:
@davidyan74 Yes, in our current use cases, only the operator main thread will use it